app/models/validations/tenancy_validations.rb

```diff
@@ -3,48 +3,67 @@ module Validations::TenancyValidations
   # or 'validate_' to run on submit as well
   include Validations::SharedValidations
 
```

Comment:
These are very similar validations, but they apply to the question as asked on different pages with slightly different criteria for how you get there and what affects the minimum value - hence splitting them out.